.github/workflows/build.yml

```
@@ -0,0 +1,52 @@
+      - name: Build application
+        run: npm run build
+        
+      - name: Verify build output
+        run: |
+          echo "Next.js build output structure:"
+          ls -la .next/ || echo ".next directory not found"
+          
+      # Docker build only on main branch pushes
+      - name: Set up Docker Buildx
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)